tfexec/show.go

```diff
@@ -146,8 +147,47 @@ func (tf *Terraform) ShowPlanFile(ctx context.Context, planPath string, opts ...
 
 }
 
-func (tf *Terraform) showCmd(ctx context.Context, mergeEnv map[string]string, args ...string) *exec.Cmd {
-	allArgs := []string{"show", "-json", "-no-color"}
+// ShowRawPlanFile reads a given plan file and outputs the plan in a
+// human-friendly, opaque format.
+func (tf *Terraform) ShowRawPlanFile(ctx context.Context, planPath string, opts ...ShowOption) (string, error) {
+	if planPath == "" {
+		return "", fmt.Errorf("planPath cannot be blank: use Show() if not passing planPath")
+	}
+
+	c := defaultShowOptions
+
+	for _, o := range opts {
+		o.configureShow(&c)
+	}
+
+	mergeEnv := map[string]string{}
+	if c.reattachInfo != nil {
+		reattachStr, err := c.reattachInfo.marshalString()
+		if err != nil {
+			return "", err
+		}
+		mergeEnv[reattachEnvVar] = reattachStr
+	}
+
+	showCmd := tf.showCmd(ctx, false, mergeEnv, planPath)
+
+	var ret bytes.Buffer
+	showCmd.Stdout = &ret
```

Comment:
This needs to use mergewriters just in case

Reply:
`mergeWriters` is already called by `tf.runTerraformCmd`, though? In fact, that's the _only_ command that calls it.